README.md

```diff
@@ -19,88 +21,99 @@ neural network developers to manage stream pipelines and their filters easily an
   - Allow any neural network developers to manage media streams fairly easily.
 
 ## Maintainers
-* MyungJoo Ham (myungjoo.ham@samsung.com)
+* [MyungJoo Ham](https://github.com/myungjoo/)
 
 ## Reviewers
-* Jijoong Moon (jijoon.moon@samsung.com)
-* Geunsik Lim (geunsik.lim@samsung.com)
-* Sangjung Woo (sangjung.woo@samsung.com)
-* Wook Song (wook16.song@samsung.com)
-* Jaeyun Jung (jy1210.jung@samsung.com)
-* Jinhyuck Park (jinhyuck83.park@samsung.com)
-* Hyoungjoo Ahn (hello.ahn@samsung.com)
-* Sewon Oh (sewon.oh@samsung.com)
+* [Jijoong Moon](https://github.com/jijoongmoon)
+* [Geunsik Lim](https://github.com/leemgs)
+* [Sangjung Woo](https://github.com/again4you)
+* [Wook Song](https://github.com/wooksong)
+* [Jaeyun Jung](https://github.com/jaeyun-jung)
+* [Jinhyuck Park](https://github.com/jinhyuck-park)
+* [Hyoungjoo Ahn](https://github.com/helloahn)
+* [Sewon Oh](https://github.com/ohsewon)
 
 ## Components
 
-Note that this project has just started and most of the components are in design phase.
+Note that this project has just started and many of the components are in design phase.
 
 ### Gstreamer Type
 
 - other/tensor
-- other/tensors (W.I.P. jijoong-moon)
-- other/tensorsave (W.I.P. myungjoo)
+- other/tensors
+- other/tensorsave (Planned)
 
 findtype specifics: refer to the wiki (other/tensorsave)
-meta specifics: W.I.P. jijoong-moon
 
 ### Gstreamer Elements (Plugins)
 
-- tensor\_converter
-  - Video
-    - Prototype for video/xraw (RGB/BGRx) is implemented.
-    - Caution: if width is not divisible by 4, RGB video incurs memcpy.
-  - Audio
-    - Planned (wooksong)
-  - Text
-    - Planned
-- tensor\_filter
-  - Main
-    - Supported
+Note that "stable" does not mean that it is complete. It means that it has enough test cases and complies with the overall design; thus, "stable" features probably won't be modified extensively. Features marked "experimental" can be modified extensively due to its incomplete design and implementation or crudeness.
```

Comment:
BTW, How about we try to put the contents in ./Documentation/ folder? If we keep a lot of statements in README.md, it will not be helpful to some users that visit this repository.

Reply:
Updated as you commented.